### PR TITLE
Adds `About Polaris` page to polaris.shopify.com

### PR DESCRIPTION
**Pull request by yurm04**
* adds `About Polaris` page to /foundations section

localhost_3000_foundations_foundations_about-polaris

### CODE REVIEW

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (yurm04) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md:77]: Don't merge this until the patterns page gets merged in!

https://github.com/Shopify/polaris/pull/6856

File: polaris.shopify.com/src/data/navItems.ts
Comment (yurm04) [inline polaris.shopify.com/src/data/navItems.ts]: We need a new icon for this card on the foundations page

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (dominikwilkowski) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]: As soon as we can we should each of the two sections to their respective documentations.

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
description: Learn what the Polaris design system is and how it’s used by designers and developers to build world-class Shopify admin experiences.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
Polaris is the design system for the Shopify admin. It’s used both internally at Shopify and externally by third-party developers and designers. From accessibility to performance to design expertise, Polaris gives you the tools to build a world-class admin experience.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
Merchants set up their store, configure settings, and manage their business using the Shopify admin. It includes core aspects of the merchant’s Shopify business, including orders, products, and customers.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
The Shopify admin is also available as iOS and Android apps. Polaris components are available as native user interface (UI) elements built with React Native. Because the iOS and Android apps also make use of embedded web views for parts of the admin, Polaris web components are used in these cases.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
Primitives are the building blocks of the Polaris design system. They’re the smallest units of the system that represent codified design decisions and reusable visual elements. The Polaris design system contains primitive tokens and primitive components that can be combined together to create admin UIs.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
Polaris components are the smallest reusable UI elements for supporting a variety of admin merchant experiences. Polaris components are one of our design system primitives, and are often referred to as primitive components.
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
Polaris tokens and components can be assembled together to solve any number of merchant needs in the admin. Compositions are the result of design tokens and/or primitives being put together to create a specific user experience. 
```

File: polaris.shopify.com/content/foundations/foundations/about-polaris/index.md
Comment (selenehinkley) [inline polaris.shopify.com/content/foundations/foundations/about-polaris/index.md]:
```suggestion
These patterns center on the merchant and how we build experiences to meet their needs.
```

**Review by selenehinkley — APPROVED** [review]
Good to go once copy edits are committed. Depending on how quickly the other reviews come in, I may take a pass with a designer to add a few points from a design lens (but this can also happen afterwards and doesn't need to be a blocker).

**Review by alex-page — COMMENTED** [review]
Hey this isn't added to the search results in the foundations.json file

**Comment by yurm04** [comment]
@alex-page ty, went back and added it!

**Review by alex-page — APPROVED** [review]

**Pull request merged**
